apps/aehttp/test/aehttp_sc_SUITE.erl

```diff
@@ -3350,6 +3391,16 @@ maybe_slogan(Map, Config) ->
             Map#{slogan => S}
     end.
 
+maybe_fee(#{fee := _} = Map, _) ->
```

Comment:
If this function stays, it should be renamed since its not a getter, but a setter.

Reply:
I see that the other setters are named in a similar fashion, so this should be addresses separately for all of these functions.